cloudsync/sync/manager.py

```diff
@@ -1052,6 +1048,8 @@ def _handle_dir_delete_not_empty(self, sync, changed, synced):
         log.info("kids exist, mark changed and punt %s", sync[changed].path)
         for kid, _ in self.state.get_kids(sync[changed].path, changed):
             kid[changed].changed = time.time()
+            # hack to ensure state.needs_sync() recognizes this as an actionable change
+            kid[changed].sync_path = "REALLY-changed"
```

Comment:
if the changed flag is useless, we should remove it or fix it right?  i think messing with the sync path could be disasterous in the wrong situations (rename conflict handling)

Reply:
added state.set_force_sync() to address this